spec/features/finance/banding_tracker_spec.rb

```diff
@@ -57,7 +57,7 @@ def generate_declarations(state:)
 
     create(:milestone,
```

Comment:
Good catch! I wonder if we can remove creating the milestone all together? I think `with_default_schedules` will create the default schedule and all the milestones we need 🤔

Reply:
Hi @cwrw I've removed creating the milestone.. looks like all good. tks for yor review. 👍 😄